sympy/solvers/tests/test_inequalities.py

```diff
@@ -218,6 +218,10 @@ def test_hacky_inequalities():
     assert reduce_inequalities(Ne(0, x - y), symbols=[x]) == Ne(x, y)
 
 
+def test_issue_10198():
+    assert reduce_inequalities(-1 + 1/abs(1/x - 1) < 0)
```

Comment:
Following the example of other lines in the file, you should provide `== result` to the right of each of these indicating what the result should be, e.g. `assert foo == bar`, otherwise they automatically pass for any non-false value.